src/Synapse/Validator/Constraints/RowExists.php

```diff
@@ -27,6 +28,18 @@ class RowExists extends Constraint
      */
     public function __construct(AbstractMapper $mapper, $options = null)
     {
+        if (isset($options['filterCallback']) && isset($options['field'])) {
+            throw new LogicException('filterCallback and field are both set. Only one is expected.');
+        }
+
+        if (isset($options['field'])) {
+            $options['filterCallback'] = function ($value) use ($options) {
+                return [
+                    $options['field'] => $value
+                ];
+            };
```

Comment:
The `$message` (or at least `$field`) needs to be modified if `filterCallback` is sent in. Not sure the best way to do this.